operator/internal/manifests/internal/config/loki-config.yaml

```
@@ -222,6 +232,48 @@ limits_config:
   shard_streams:
     enabled: true
     desired_rate: {{ . }}MB
+{{- end }}
+{{- with .Stack.Limits.Global.OTLP }}
+  otlp_config:
+    {{- with .ResourceAttributes }}
+    resource_attributes:
+      ignore_defaults: {{ .IgnoreDefaults }}
+      {{- with .Attributes}}
+      attributes_config:
+      {{- range . }}
+      - action: {{ .Action }}
+        {{- with .Regex }}
+        regex: {{ . }}
+        {{- end }}
+        attributes:
+        {{- range .Attributes }}
+        - {{ . }}
+        {{- end}}
+      {{- end }}
+      {{- end }}
+    {{- end }}
+    {{- with .ScopeAttributes }}
+    scope_attributes:
+    {{- range . }}
+    - action: {{ .Action }}
+      attributes:
+      {{- range .Attributes }}
+      - {{ . }}
+      {{- end }}
+      regex: {{ .Regex }}
+    {{- end }}
+    {{- end }}
+    {{- with .LogAttributes }}
+    log_attributes:
+    {{- range . }}
+    - action: {{ .Action }}
+      attributes:
+      {{- range .Attributes }}
+      - {{ . }}
+      {{- end }}
+      regex: {{ .Regex }}
```

Update from the previous thread:

```suggestion
      {{- with .Regex }}
      regex: {{ . }}
      {{- end }}
      {{- with .Attributes }}
      attributes:
      {{- range . }}
      - {{ . }}
      {{- end }}
      {{- end }}
```